### PR TITLE
Add absolute import checking

### PR DESCRIPTION
**Pull request by mvsusp**
*Issue #, if available:*
https://github.com/aws/sagemaker-python-sdk/issues/179
https://github.com/aws/sagemaker-python-sdk/issues/139

*Description of changes:*
- enforcing absolute importing so Python 2.7 prioritizes absolute imports.
- fix for tensorflow-serving-api issues related to multiple versions.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/tensorflow/__init__.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
 import sys
 
 import os
```
Comment (laurenyu) [inline src/sagemaker/tensorflow/__init__.py]: while you're here, want to also group `os` and `sys`?

File: doc/conf.py [hunk 2]
```diff
@@ -1,4 +1,17 @@
-# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (laurenyu) [inline doc/conf.py]: is there a reason we have 2017-2018 for some and just 2018 for others?

File: tests/component/__init__.py [hunk 3]
```diff
@@ -10,3 +10,4 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+from __future__ import absolute_import
```
Comment (laurenyu) [inline tests/component/__init__.py]: do we need this for files with no code?

File: doc/conf.py (hunk 2 above)
Comment (mvsusp) [reply doc/conf.py]: Not really, I will update all the copyright to the 2017-2018 range.

**Pull request merged**